jvm/src/main/kotlin/app/cash/trifle/SignedData.kt

```diff
@@ -36,6 +36,9 @@ data class SignedData internal constructor(
     return isVerified
   }
 
+  fun verifyAndExtract(certAnchor: Certificate): VerifiedData? =
+    if (verify(certAnchor)) VerifiedData(envelopedData.data, certificates) else null
```

Comment:
I think we can have an extension method such as
```
fun SignedData.toVerifiedData: VerifiedData = 
...
```

Reply:
I'm going to keep this as-is given the rest of the change, and we can update it later if necessary.  I'm not seeing the advantage to using the method extension vs. current approach.